src/BinSkim.Driver/BinSkim.Driver.csproj

```diff
@@ -65,7 +65,7 @@
     <MsRuntime Include="..\..\refs\runtime\*" />
   </ItemGroup>
   <Target Name="CopyMsDiaLibs" AfterTargets="build">
-    <Copy SourceFiles="@(MsDiaLibs)" DestinationFolder="$(OutputPath)\" />
+	<Copy SourceFiles="@(MsDiaLibs)" DestinationFolder="$(OutputPath)\net9.0\" />
```

Comment:
>net9 [](http://example.com/codeflow?start=67&length=4)

this change has broken baselining, the command-line tool is no longer loading files from this location when running the update script.